hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/BasicCustomCallOperation.java

```diff
@@ -91,6 +93,11 @@ default Operation.OperationResult executeChecked(@NonNull final MessageFrame fra
         requireNonNull(evm);
         requireNonNull(frame);
         try {
+            final long cost = cost(frame);
```

Comment:
(Sigh.  This `cost()` takes a `Frame` whereas the original Besu `cost()` takes a boolean.  Especially noticeable in this particular PR where _both_ are used! Can't change the Besu one, so wish this one had a different name ... darn.)